lib/expired_record_cleaner.rb

```diff
@@ -0,0 +1,53 @@
+class ExpiredRecordCleaner
+  attr_reader :verbose, :fiscal_year_start
+
+  def initialize(dtim, ok_to_act = false, verbose = false)
+    @fiscal_year_start = self.class.fiscal_year_start(dtim)
+    @ok_to_act = ok_to_act
+    @verbose = verbose
+  end
+
+  def vacuum_old_proposals
+    proposals = Proposal.pending.where("created_at < '#{@fiscal_year_start}'")
+    ids = []
+    proposals.each do |proposal|
```

Comment:
what about using this type of syntax to make sure the action removes one proposal and then also checking that the proposal created in the test no longer exists? 

https://github.com/18F/C2/blob/a91e65046e0295b13423a26edf716040446678c8/spec/features/roles_conversion_spec.rb#L11